docs/upgrade-guides/upgrading-nextjs-from-v4-to-v5.mdx

```diff
@@ -25,191 +25,126 @@ To aid you in this upgrade, Clerk built a CLI called [`@clerk/upgrade`](https://
 
-<Accordion titles={["Changes to appearance prop", "Changes to elements appearance property", "Changes to localization"]} heading="h3">
-  <AccordionPanel>
-    **Note:** If you want your components to appear exactly as the Clerk team designed them you can skip this section.
+### Per-route auth config
 
-    The default values of some [appearance variables](/docs/components/customization/variables) have changed which may impact your UI (if you are not already overriding them).
+Sometimes, code speaks louder than words. Let's get right into an example:
 
-    - The default `colorPrimary` value changed from `#103FEF` to `#2F3037`. As the new color is a dark grey, the `colorPrimary` of the dark theme was changed to `#FFFFFF`.
-    - The default `fontSize` value changed from `1rem` to `0.8125rem`
-    - The default `fontWeight` values changed from `{ normal: 400, medium: 500, bold: 600 }` to `{ normal: 400, medium: 500, bold: 700 }`
-    - Previously, the default value for `fontSmoothing` was `auto`. This value is now unset. If you want to pass a custom value to it, you can still do so.
-  </AccordionPanel>
-  <AccordionPanel>
-    **Note:** If you're not using the [`elements` appearance property](/docs/components/customization/overview#use-custom-css-classes-to-style-clerk-components) you can skip this section.
-
-    You're able to pass additional classes to Clerk component elements by using the `elements` property on `appearance`. Some keys have changed or have been completely removed. Clerk recommends using IntelliSense in your IDE to inspect the type of `appearance.elements`.
-
-    Here are some more details on individual component changes.
+```ts filename="app/src/dashboard/layout.tsx"
+import { auth } from '@clerk/nextjs/server'
 
-    #### `<SignIn />` and `<SignUp />`
+export default async function DashboardLayout({ children }) {
+    auth().protect();
+    // 👆 This one line is all it takes - the layout and all its children
+    // will now redirect to your sign in page for un-authenticated users.
 
-    Change the appearance of these elements if it does not match the style of your application:
-
-    - `cardBox` (or `cl-cardBox` class). This element is the container of Clerk's cards.
-    - `footer`. The footer is now slightly darker than the card's content.
-
-    #### `<UserButton />`
+    return <>{children}</>
+}
+```
 
-    The `userButtonTrigger` now encapsulates the `avatarBox` which may impact the styling of your `<UserButton />`. If you have enabled multi-session handling you will also need to re-customize parts of the `<UserButton />` component.
+In this example, we add `auth().protect()` to a layout, and like magic, that layout and all of it's children will be gated by auth.
 
-    #### `<UserProfile />` and `<OrganizationProfile />`
+Want to get more specific with permission gating? It's just as simple:
```

Comment:
https://github.com/clerk/clerk-docs/blob/main/STYLEGUIDE.md#31-do-not-assume-proficiency

It might not be simple for everyone. So when they read it, don't understand, and will be left with a bad feeling.